packages/components/src/popper/Popper.tsx

```diff
@@ -150,6 +150,7 @@ class Popper extends Component<PopperProps, PopperState> {
     let { options } = this.props;
     options = {
       placement: 'auto',
+      modifiers: { preventOverflow: { boundariesElement: 'viewport' } },
```

Comment:
It's part of that ticket. Binds to viewport. Didn't change the placement from auto though. Don wanted to check that it didn't cause overflow issues by changing default to bottom instead of auto